test/python/tensor_layout.py

```diff
@@ -18,7 +18,7 @@ def getTensorMemoryLayout(memorySpace):
     ):
         return tt.TensorMemoryLayout.Interleaved
     else:
-        return tt.TensorMemoryLayout.NoneLayout
+        return tt.TensorMemoryLayout.None_
```

Comment:
Hey Nick thank you this looks great! I used 'NoneLayout' because I didn't want to do `.None` here. Probably a stupid question but how does this `::None` propagate to `.None_` here?

Reply:
Since `None` is a keyword, it'd be illegal to declare a member with the name None, so tablegen just adds an underscore at the end.